cmd/db.go

```diff
@@ -47,7 +54,19 @@ var dbCmd = &cobra.Command{
 			var source string
 			for rows.Next() {
 				rows.Scan(&txnId, &txnDate, &item, &cost, &location, &category, &source)
-				table.Append([]string{txnId, txnDate, item, fmt.Sprintf("%f", cost), location, category, source})
+				table.Append([]string{txnId, txnDate, location, category, source, item, fmt.Sprintf("%.2f", cost)})
+			}
+
+			if showTotal {
+				if rows, err := database.Query(totalStatementSQL); err != nil {
+					logger.Warnf("Unable to calculate the total: %v", err)
```

Comment:
Should this be a `Fatalf` instead?

Reply:
My thought was that we can still show the user their transactions even when unable to calculate their total, show them a warning for that instead.
I could change this to Fatal.

Reply:
Since the user is explicity setting `--total` to view the total, it seems counterintutive to show them the data without totals. I think it might be better to make it Fatal.